pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -251,8 +251,10 @@ def check_set_errors(self):
     def ask(self, command, query_delay=0):
         """Send a command and read the response, stripping white spaces.
 
-        Usually the properties use the ``values()`` method that adds a strip call,
-        however several methods use directly the result from ask to be casted into some other types.
+        Usually the properties use the
+        :meth:`values() <pymeasure.instruments.common_base.CommonBase.values>`

```

Comment:
sphinx will show `values()` if you write the following:
```suggestion
        :meth:`pymeasure.instruments.common_base.CommonBase.values`
```

Reply:
Sorry, I forgot the tilde `~`:
```suggestion
        :meth:`~pymeasure.instruments.common_base.CommonBase.values`
```
That will abbreviate it.

Reply:
Thanks, just fixed that. There are like 10 ways of doing the same thing in RST.